lib/parse-args.js

```diff
@@ -191,18 +189,13 @@ function buildYargs (withCommands = false) {
 function hideInstrumenterArgs (yargv) {
   let argv = process.argv.slice(1)
   argv = argv.slice(argv.indexOf(yargv._[0]))
-  if (argv[0][0] === '-') {
```

Comment:
Not sure what use case this was addressing.  From what I can tell, if the first character of the first argument is a `-` then add the path of where C8 is located to the end of the argument array.  What's the purpose of this?  Do I need to add a Test Case instead of removing the code.